src/components/portfolio/DashboardTable.tsx

```diff
@@ -78,7 +78,7 @@ const DashboardTableRow: React.FC<DashboardTableRowProps> = ({
     >
       <TableCell align="center">
         {'score' in portfolio && (
-          <PortfolioScore score={portfolio.score} dashboard />
+          <PortfolioScore score={Math.round(portfolio.score)} dashboard />
```

Comment:
Wouldn't it be a better idea to round the score in the `PortfolioScore` component?

Reply:
Yeah true, I didn't notice. Thanks for pointing that out!